apps/extension/src/pages/tx-result/success.tsx

```diff
@@ -0,0 +1,88 @@
+import { observer } from "mobx-react-lite";
+import React, { FunctionComponent, useEffect, useRef } from "react";
+import styled from "styled-components";
+import lottie from "lottie-web";
+import AniSuccess from "../../public/assets/lottie/tx-result/success.json";
+import { Stack } from "../../components/stack";
+import { Gutter } from "../../components/gutter";
+import { H3, Subtitle2 } from "../../components/typography";
+import { useIntl } from "react-intl";
+import { ColorPalette } from "../../styles";
+import { Box } from "../../components/box";
+import { useSearchParams, useNavigate } from "react-router-dom";
+
+export const TxResultSuccessPage: FunctionComponent = observer(() => {
+  const intl = useIntl();
+  const animDivRef = useRef<HTMLDivElement | null>(null);
+  const navigate = useNavigate();
+
+  const [searchParams] = useSearchParams();
+
+  const isFromEarnTransfer = searchParams.get("isFromEarnTransfer");
+
+  useEffect(() => {
+    if (animDivRef.current) {
+      const anim = lottie.loadAnimation({
+        container: animDivRef.current,
+        renderer: "svg",
+        loop: true,
+        autoplay: true,
+        animationData: AniSuccess,
+      });
+
+      return () => {
+        anim.destroy();
+      };
+    }
+  }, []);
+
+  useEffect(() => {
+    if (isFromEarnTransfer) {
+      setTimeout(() => {
+        navigate(`/earn/amount`);
```

Comment:
마이너한 케이스긴 한데 Success하고나서 /earn/amount page로 다시 가게 되었을 때 back 버튼 클릭하면 다시 Success로 가더라고요. navigate할 때 param 하나 보내서 백버튼을 비노출하거나 클릭 시 home으로 보내주는게 어떨까 합니다!

Reply:
백버튼 클릭시 home로 보내주도록 변경했습니다!

Reply:
여기서 param 넣어줘야할 거 같아요.
```suggestion
        navigate(`/earn/amount?isFromEarnTransfer=true`);
```

Reply:
저장을 안하는 실수를;; 감사합니다!